### PR TITLE
Aden 3867

### PR DESCRIPTION
**Pull request by jacekjursza**
Remove Lock/Unlock CSS functionality

### CODE REVIEW

**Comment by WikiaTech** [comment]
SonarQube analysis reported 9 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [WikiaSkin.class.php#L370](https://github.com/Wikia/app/blob/4762df107b983a6be911aaa14e31150ad8f8d5d7/includes/wikia/nirvana/WikiaSkin.class.php#L370): Remove this unused "$wf" local variable. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)
1. MINOR [WikiaSkin.class.php#L115](https://github.com/Wikia/app/blob/4762df107b983a6be911aaa14e31150ad8f8d5d7/includes/wikia/nirvana/WikiaSkin.class.php#L115): Define a constant instead of duplicating this literal "&amp;" 4 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
1. MINOR [WikiaSkin.class.php#L345](https://github.com/Wikia/app/blob/4762df107b983a6be911aaa14e31150ad8f8d5d7/includes/wikia/nirvana/WikiaSkin.class.php#L345): Remove this method "buildPersonalUrls" to simply inherit it. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1185)
1. MINOR [WikiaSkin.class.php#L349](https://github.com/Wikia/app/blob/4762df107b983a6be911aaa14e31150ad8f8d5d7/includes/wikia/nirvana/WikiaSkin.class.php#L349): Remove this method "buildContentNavigationUrls" to simply inherit it. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1185)
1. MINOR [WikiaSkin.class.php#L353](https://github.com/Wikia/app/blob/4762df107b983a6be911aaa14e31150ad8f8d5d7/includes/wikia/nirvana/WikiaSkin.class.php#L353): Remove this method "buildContentActionUrls" to simply inherit it. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1185)
1. MINOR [WikiaSkin.class.php#L357](https://github.com/Wikia/app/blob/4762df107b983a6be911aaa14e31150ad8f8d5d7/includes/wikia/nirvana/WikiaSkin.class.php#L357): Remove this method "buildNavUrls" to simply inherit it. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1185)
1. MINOR [WikiaSkinTest.php#L98](https://github.com/Wikia/app/blob/4762df107b983a6be911aaa14e31150ad8f8d5d7/includes/wikia/tests/WikiaSkinTest.php#L98): Define a constant instead of duplicating this literal "jquery" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
1. INFO [WikiaSkin.class.php#L218](https://github.com/Wikia/app/blob/4762df107b983a6be911aaa14e31150ad8f8d5d7/includes/wikia/nirvana/WikiaSkin.class.php#L218): Complete the task associated to this "TODO" comment. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)
1. INFO [WikiaSkinTest.php#L44](https://github.com/Wikia/app/blob/4762df107b983a6be911aaa14e31150ad8f8d5d7/includes/wikia/tests/WikiaSkinTest.php#L44): Move this trailing comment on the previous empty line. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)

File: includes/wikia/nirvana/WikiaSkin.class.php [hunk 1]
```diff
@@ -278,13 +278,10 @@ public function getStylesWithCombinedSASS(Array &$sassFiles) {
 		// get a single URL to fetch all the required SASS files
 		$sassFilesUrl = $this->assetsManager->getSassesUrl($sassFiles);
 
-		// recovery unlock css
-		$unlockUrl = ARecoveryUnlockCSS::getUnlockCSSUrl();
-
 		wfDebug( sprintf( "%s: combined %d SASS files\n", __METHOD__, count($sassFiles) ) );
 
 		wfProfileOut(__METHOD__);
-		return Html::linkedStyle($sassFilesUrl) . Html::linkedStyle($unlockUrl) . implode('', $cssLinks);
```
Comment (fraszczakszymon) [inline includes/wikia/nirvana/WikiaSkin.class.php:287]: I think we should split this PR. It may cause blocked content in first 24h. Am I right?

**Comment by fraszczakszymon** [comment]
There are other references:
https://github.com/Wikia/app/blob/dev/extensions/wikia/AssetsManager/config.php#L872
https://github.com/Wikia/app/blob/dev/extensions/wikia/ARecoveryEngine/ARecoveryModule.class.php#L4
https://github.com/Wikia/app/blob/dev/extensions/wikia/ARecoveryEngine/ARecoveryModule.class.php#L29

**Comment by WikiaTech** [comment]
SonarQube analysis reported 9 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. MAJOR [WikiaSkin.class.php#L370](https://github.com/Wikia/app/blob/18a539ac0a1a9575ef14ffb6263c159ed724963f/includes/wikia/nirvana/WikiaSkin.class.php#L370): Remove this unused "$wf" local variable. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)
1. MINOR [WikiaSkin.class.php#L115](https://github.com/Wikia/app/blob/18a539ac0a1a9575ef14ffb6263c159ed724963f/includes/wikia/nirvana/WikiaSkin.class.php#L115): Define a constant instead of duplicating this literal "&amp;" 4 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
1. MINOR [WikiaSkin.class.php#L345](https://github.com/Wikia/app/blob/18a539ac0a1a9575ef14ffb6263c159ed724963f/includes/wikia/nirvana/WikiaSkin.class.php#L345): Remove this method "buildPersonalUrls" to simply inherit it. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1185)
1. MINOR [WikiaSkin.class.php#L349](https://github.com/Wikia/app/blob/18a539ac0a1a9575ef14ffb6263c159ed724963f/includes/wikia/nirvana/WikiaSkin.class.php#L349): Remove this method "buildContentNavigationUrls" to simply inherit it. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1185)
1. MINOR [WikiaSkin.class.php#L353](https://github.com/Wikia/app/blob/18a539ac0a1a9575ef14ffb6263c159ed724963f/includes/wikia/nirvana/WikiaSkin.class.php#L353): Remove this method "buildContentActionUrls" to simply inherit it. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1185)
1. MINOR [WikiaSkin.class.php#L357](https://github.com/Wikia/app/blob/18a539ac0a1a9575ef14ffb6263c159ed724963f/includes/wikia/nirvana/WikiaSkin.class.php#L357): Remove this method "buildNavUrls" to simply inherit it. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1185)
1. MINOR [WikiaSkinTest.php#L98](https://github.com/Wikia/app/blob/18a539ac0a1a9575ef14ffb6263c159ed724963f/includes/wikia/tests/WikiaSkinTest.php#L98): Define a constant instead of duplicating this literal "jquery" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
1. INFO [WikiaSkin.class.php#L218](https://github.com/Wikia/app/blob/18a539ac0a1a9575ef14ffb6263c159ed724963f/includes/wikia/nirvana/WikiaSkin.class.php#L218): Complete the task associated to this "TODO" comment. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)
1. INFO [WikiaSkinTest.php#L44](https://github.com/Wikia/app/blob/18a539ac0a1a9575ef14ffb6263c159ed724963f/includes/wikia/tests/WikiaSkinTest.php#L44): Move this trailing comment on the previous empty line. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)

**Comment by jacekjursza** [comment]
public static function isLockEnabled() actually should stay there - it's still used in Oasis_Index and needed by JS bootstrap

**Review by fraszczakszymon — APPROVED** [review]
LGTM 👍

**Comment by WikiaTech** [comment]
SonarQube analysis reported 9 issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:
1. MAJOR [WikiaSkin.class.php#L370](https://github.com/Wikia/app/blob/bb1ca22448036bb087b594a495eeb65e7a4783b8/includes/wikia/nirvana/WikiaSkin.class.php#L370): Remove this unused "$wf" local variable. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1481)
2. MINOR [WikiaSkin.class.php#L115](https://github.com/Wikia/app/blob/bb1ca22448036bb087b594a495eeb65e7a4783b8/includes/wikia/nirvana/WikiaSkin.class.php#L115): Define a constant instead of duplicating this literal "&amp;" 4 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
3. MINOR [WikiaSkin.class.php#L345](https://github.com/Wikia/app/blob/bb1ca22448036bb087b594a495eeb65e7a4783b8/includes/wikia/nirvana/WikiaSkin.class.php#L345): Remove this method "buildPersonalUrls" to simply inherit it. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1185)
4. MINOR [WikiaSkin.class.php#L349](https://github.com/Wikia/app/blob/bb1ca22448036bb087b594a495eeb65e7a4783b8/includes/wikia/nirvana/WikiaSkin.class.php#L349): Remove this method "buildContentNavigationUrls" to simply inherit it. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1185)
5. MINOR [WikiaSkin.class.php#L353](https://github.com/Wikia/app/blob/bb1ca22448036bb087b594a495eeb65e7a4783b8/includes/wikia/nirvana/WikiaSkin.class.php#L353): Remove this method "buildContentActionUrls" to simply inherit it. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1185)
6. MINOR [WikiaSkin.class.php#L357](https://github.com/Wikia/app/blob/bb1ca22448036bb087b594a495eeb65e7a4783b8/includes/wikia/nirvana/WikiaSkin.class.php#L357): Remove this method "buildNavUrls" to simply inherit it. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1185)
7. MINOR [WikiaSkinTest.php#L98](https://github.com/Wikia/app/blob/bb1ca22448036bb087b594a495eeb65e7a4783b8/includes/wikia/tests/WikiaSkinTest.php#L98): Define a constant instead of duplicating this literal "jquery" 3 times. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
8. INFO [WikiaSkin.class.php#L218](https://github.com/Wikia/app/blob/bb1ca22448036bb087b594a495eeb65e7a4783b8/includes/wikia/nirvana/WikiaSkin.class.php#L218): Complete the task associated to this "TODO" comment. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)
9. INFO [WikiaSkinTest.php#L44](https://github.com/Wikia/app/blob/bb1ca22448036bb087b594a495eeb65e7a4783b8/includes/wikia/tests/WikiaSkinTest.php#L44): Move this trailing comment on the previous empty line. [rule](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)

**Pull request merged**